wp-content/themes/oceanwp-child/bbpress/content-archive-topic.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * Archive Topic Content Part
+ *
+ * @package bbPress
+ * @subpackage Theme
+ */
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+?>
+
+<style>
+	.page-header{
+		display:none;
+	}
+
+	.content-area{
+		padding-right: 0px;
+    	border-right-width: 0px;
+	}
+</style>
+
+<div class="row mt-60">
+	<div class="col-md-4 col-lg-4 col-sm-12">
+		<h2>Discuss</h2>
+		<p>The Global Green Recovery Collaborative invites members to share ideas for a green recovery and working together, 
+			to guide one another and share resources. Connect with experts, practitioners, activists, and researchers.</p>
+		<p>Register and create your profile to participate. Frequently Asked Questions and Community Guidelines will 
+			answer any questions you have about joining and participating.</p>
+
+		<div class="row">
+			<div class="col-md-12 col-lg-12 col-sm-12">
+				<a href="#" class="initiative-website">Community guidelines</a>
+			</div>
+			<div class="col-md-12 col-lg-12 col-sm-12 mt-30">
+				<a><button class="ggrc-btn-white-lg">Log In</button></a>
```

Comment:
Button that does nothing.

Reply:
Done as per design, will be linked once the log in page is done.